WordPress/twentytwentyfour
# Add RSVP pattern

**Description**

Alternate approach to https://github.com/WordPress/twentytwentyfour/pull/317, using a similar layout implemented in the about pattern. 

Closes https://github.com/WordPress/twentytwentyfour/issues/122.

**Screenshots**

https://github.com/WordPress/twentytwentyfour/assets/1813435/e184f269-533b-4cbc-8eee-605895340181


**Testing Instructions**


1. Search for `rsvp` in the pattern inserter. 
2. Add it to the page. 
3. Check it on mobile. 

**Contributors**

@carolinan
